### PR TITLE
Update buttons/fonts to use correct font-family + change `btn-primary` color

### PR DESCRIPTION
**Pull request by joshdarby**
## Changes

This pull request makes the following changes:

- Updates `body`, `button`, `input`, and miscellaneous header and footer elements to have `font-family:@body;` which is `open sans`.
- Updates `.btn-primary` background color to be `#d24f2f;`

## Why

For #53 / #54 

## Testing/Questions

Features that this PR affects:

- Typography styles
- Button colors

Questions that need to be answered before merging:

- [x] Is this PR targeting the correct branch in this repository?
- [ ] Are there any other elements that need their fonts updated?

### CODE REVIEW

File: wp-content/themes/borderzine/less/typography.less [hunk 1]
```diff
@@ -9,6 +10,15 @@ a {
...
 }
 
+button {
+  font-family: @body;
+}
+
+.btn-primary,
+.btn-primary:hover {
+  background-color: @orange;
+}
```
Comment (benlk) [inline wp-content/themes/borderzine/less/typography.less]: We need to have some visual difference here for the :hover and :focus states from the inactive state. Can we use a different background color for that, or a white border on the element?

Comment (joshdarby) [reply wp-content/themes/borderzine/less/typography.less]: @benlk https://github.com/INN/umbrella-borderzine/pull/62/commits/09fae219522e02863bf219ca7690b674fcd0f282 uses `#2e4052` as the `btn-primary:focus` background color. `#2e4052` is the color of their donate button (that turns orange on hover) on their prod site.

Comment (benlk) [reply wp-content/themes/borderzine/less/typography.less]: Why not also use that color for `:hover`?

Comment (joshdarby) [reply wp-content/themes/borderzine/less/typography.less]: @benlk a few days ago @kaylima mentioned the hover state background color for buttons was fine as-is, so I didn't want to change it.

**Pull request merged**